.github/workflows/package.yml

```diff
@@ -237,6 +237,7 @@ jobs:
     env:
       APACHE_ARROW_REPOSITORY: ${{ github.workspace }}/apache-arrow
     runs-on: ubuntu-latest
+    timeout-minutes: ${{ (matrix.id == 'debian-trixie-arm64' && 270) || 90 }}
```

Comment:
I make sure that this setting works as follows.
```diff
git show
commit 4938337dbb9e7f63262b7232236aa1a5a4a75c0d (HEAD -> set-timeout-in-build-step, origin/set-timeout-in-build-step)
Author: otegami <otegami@clear-code.com>
Date:   Wed Aug 21 13:53:16 2024 +0900

    debug

diff --git a/.github/workflows/package.yml b/.github/workflows/package.yml
index c33a87a11..4f39ba86f 100644
--- a/.github/workflows/package.yml
+++ b/.github/workflows/package.yml
@@ -237,7 +237,7 @@ jobs:
     env:
       APACHE_ARROW_REPOSITORY: ${{ github.workspace }}/apache-arrow
     runs-on: ubuntu-latest
+    timeout-minutes: ${{ (matrix.id == 'debian-trixie-arm64' && 10) || 5 }}
     steps:
       - uses: actions/checkout@v4
         with:
```

![Screenshot from 2024-08-21 14-12-35](https://github.com/user-attachments/assets/abcc42b2-4805-49b7-adeb-587bcc1fbf58)

ref: https://github.com/groonga/groonga/actions/runs/10483732563/usage?pr=1886